src/rendering/webgl/programs/edge.arrowHead.ts

```diff
@@ -102,6 +80,6 @@ export default class EdgeArrowHeadProgram extends EdgeProgram<typeof UNIFORMS[nu
     gl.uniform1f(u_sizeRatio, params.sizeRatio);
     gl.uniform1f(u_correctionRatio, params.correctionRatio);
 
-    gl.drawArrays(gl.TRIANGLES, 0, this.verticesCount);
+    this.drawWebGL(gl.TRIANGLES);
```

Comment:
I think we discussed this but maybe the gl method can be a property of the class because we won't call drawWebGL with alternating ones at runtime?

Reply:
Branche [v3-1370-1371](https://github.com/jacomyal/sigma.js/tree/v3-1370-1371) does that indeed.